src/server/rpc/procedures/iex-trading/iex-trading.js

```diff
@@ -0,0 +1,130 @@
+    return this._requestData(queryOpts)
+        .then(body => {
+            return body[company];
+        })
+};
+
+/**
+ * Get the last opening price of the specified company
+ * @param {companySymbol} Symbol. The stock ticker symbol of the desired company
+ * @returns {double} Value of the last opening price for that stock
+ */
+StockConsumer.lastOpenPrice = function(companySymbol) {
+	// Defines the query options for the API, including the user-defined value to receive
+
+    queryOpts.queryString = '';
+    queryOpts.queryString = '/stock/' + companySymbol + '/quote';
+
+    company = 'open';
+
+	// Return statement, using a promise to allow _requestData to receive the value from the API before returning
+    return this._requestData(queryOpts)
+        .then(body => {
+            return body[company];
+	    })
+};
+
+/**
+ * Get the last close price of the specified company
+ * @param {companySymbol} Symbol. The stock ticker symbol of the desired company
+ * @returns {double} Value of the last close price for that stock
+ */
+StockConsumer.lastClosePrice = function(companySymbol) {
+	// Defines the query options for the API, including the user-defined value to receive
+
+    queryOpts.queryString = '';
+    queryOpts.queryString = '/stock/' + companySymbol + '/quote';
+
+    company = 'close';
+
+	// Return statement, using a promise to allow _requestData to receive the value from the API before returning
+    return this._requestData(queryOpts)
+        .then(body => {
+            return body[company];
+        })
+};
+
+/**
+ * Get a 2-n list of all the information about the specified company
+ * @param {companySymbol} Symbol. The stock ticker symbol of the desired company
+ * @returns {list} List of all the company information
+ */
+StockConsumer.companyInformation = function(companySymbol) {
+	// Defines the query options for the API, including the user-defined value to receive
+
+    queryOpts.queryString = '';
+    queryOpts.queryString = '/stock/' + companySymbol + '/quote';
+
+	// Return statement, using a promise to allow _requestData to receive the value from the API before returning
+    return this._requestData(queryOpts, stockParser)
+};
+
+	/**
+ 	* Get the daily change percent for a company
+ 	* @param {companySymbol} Symbol. The stock ticker symbol of the desired company
+ 	* @returns {double} The daily change percent for the company
+ 	*/
+StockConsumer.dailyPercentChange = function(companySymbol) {
+	// Defines the query options for the API, including the user-defined value to receive
+
+    queryOpts.queryString = '';
+    queryOpts.queryString = '/stock/' + companySymbol + '/quote';
+
+    company = 'changePercent';
+
+	// Return statement, using a promise to allow _requestData to receive the value from the API before returning
+    return this._requestData(queryOpts)
+        .then(body => {
+            return body[company];
+        })
+    };
+
+
```

Comment:
Add `StockConsumer.serviceName = 'IEXTrading';` here to fix the capitalization of the service on the client.